pkgs/applications/virtualization/virtualbox/guest-additions/default.nix

```diff
@@ -11,7 +11,7 @@ let
   # (not via videoDrivers = ["vboxvideo"]).
   # It's likely to work again in some future update.
   xserverABI = let abi = xserverVListFunc 0 + xserverVListFunc 1;
-    in if abi == "119" || abi == "120" then "118" else abi;
+    in if abi == "119" || abi == "120" || abi == "211" then "118" else abi;
```

Comment:
Nit:
```suggestion
    in if builtins.elem abi [ "119" "120" "211" ] then "118" else abi;
```

Reply:
Actually we might drop it.  Nothing but >= 1.18 is in use for quite some time, and it doesn't seem like it will change soon.  (meaning I see nothing newer on virtualbox side etiher)